assets/pages/weekly-subscription-landing/components/weeklyForm.jsx

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
this file is a bit half-baked but so is the presentation of the form! It will be nicer once the form looks nicer